public/assets/scss/_navbar.scss

```diff
@@ -0,0 +1,149 @@
+%heading-font-stack {
+    color: $blue;
+    font-size: 3rem;
+}
+%heading-text-font-stack {
```

Comment:
This is not used anywhere in the code. Can we remove it, please?

Reply:
it is